src/store/slice/room.ts

```diff
@@ -16,8 +16,14 @@ export const roomSlice = createSlice({
     setRoom: (state, action: PayloadAction<Room>) => {
       return action.payload
     },
+    setRoomNewMember(state, action: PayloadAction<WsRoomNewMemberEventBody>) {
+      state.capacity = action.payload.capacity
+      state.members = action.payload.members
+      state.hostId = action.payload.hostId
+      return state
```

Comment:
スプレッド演算子のほうが拡張性あるし 上と表記もあって嬉しい鴨試練
```suggestion
    setRoomNewMember(state, action: PayloadAction<WsRoomNewMemberEventBody>) {
      return { ...state, ...action.payload }
```